src/components/organisms/Footer/index.tsx

```diff
@@ -1,10 +1,39 @@
 import { Grid } from "@mui/material";
 import React from "react";
-import { customStyles } from "../../../theme";
 import Typo from "../../atoms/Typo/index";
 import Buttons from "../../atoms/Buttons/Buttons";
 import { KeyboardArrowDown } from "@mui/icons-material";
+import { makeStyles } from "@material-ui/core";
 
+var customStyles = makeStyles({
+  footerElement: {
+    width: "83px",
+    height: "22px",
+    fontFamily: "Graphik",
+    fontStyle: "normal",
+    fontWeight: 400,
+    fontSize: "16px",
+    lineHeight: "22px",
+    color: "#0052FF",
+  },
+
+  footerElement2: {
+    width: "83px",
+    height: "22px",
+    fontFamily: "Graphik",
+    fontStyle: "normal",
+    fontWeight: 400,
+    fontSize: "16px",
+    lineHeight: "22px",
+    color: "#343446",
+  },
```

Comment:
since both footerElement and footerElement2 are mostly same you can have a single footer property and only have separate properties for colors.